projects/libs/core-library-typescript/src/test-kit/transaction-builders.ts

```diff
@@ -22,7 +22,7 @@ export function aDummyTransactionSet(numberOfTransactions = 3): types.Transactio
   const transactions: types.Transaction[] = [];
   for (let i = 0; i < numberOfTransactions; i++) {
     const transaction = aDummyTransaction({senderPublicKey: createHash("sha256").update(`address${i}`).digest()});
-    const txHash = TransactionUtils.calculateTransactionHash(transaction);
+    const txHash = new TransactionHelper(transaction).calculateHash();
```

Comment:
Why not have an object for `Transaction`?

Reply:
It's confusing since we have a Transaction as a protobuf-derived interface. That was actually how it was called initially